cmd/openstack-cloud-controller-manager/main.go

```diff
@@ -44,7 +44,7 @@ import (
 )
 
 func main() {
-	rand.Seed(time.Now().UnixNano())
+	rand.Seed(time.Now().UnixNano()) //nolint:staticcheck
```

Comment:
I was curious:

```
cmd/openstack-cloud-controller-manager/main.go:47:2: SA1019: rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: Programs that call Seed and then expect a specific sequence of results from the global random source (using functions such as Int) can be broken when a dependency changes how much it consumes from the global random source. To avoid such breakages, programs that need a specific result sequence should use NewRand(NewSource(seed)) to obtain a random generator that other packages cannot access. (staticcheck)
        rand.Seed(time.Now().UnixNano())
```

Could we just remove this. We don't use math/rand directly in cloud-provider-openstack except in `tests/sanity/cinder/fakecloud.go`. Libraries should either be initialising this themselves or more likely using crypto/rand instead. @pierreprinetti might have an opinion on this.

That might be better as a separate PR though just in case it had unintended consequences.

Reply:
removed